src/vse_sync_pp/parsers/dpll.py

```diff
@@ -7,7 +7,7 @@
 
 from .parser import Parser
 
-class PhaseOffset(Parser):
+class TimeErrorParser(Parser):
     """Parse phase offset from a dpll CSV sample"""
     id_ = 'dpll/phase-offset'
     elems = ('timestamp', 'eecstate', 'phasestate', 'phaseoffset')
```

Comment:
Is `phaseoffset` still correct? Or do we have another `terror`?

Reply:
terror FTW